src/views/menus/AccountMenu.tsx

```diff
@@ -208,7 +208,7 @@ export const AccountMenu = () => {
                   asset={DydxChainAsset.USDC}
                   complianceState={complianceState}
                   dispatch={dispatch}
-                  hasBalance={MustBigNumber(usdcBalance).gt(0)}
+                  hasBalance={MustBigNumber(freeCollateral?.current ?? 0).gt(0)}
```

Comment:
nit -- can we just do `MustBigNumber(freeCollateral?.current).gt(0)`?, assuming must big number just converts undefined/null to 0?